### PR TITLE
New packages: Bear-2.2.1, sound-icons-0.1, dex-0.7

### PR DESCRIPTION
**Pull request by ackalker**
These are my first few contributions to Void Linux, I hope that these packages are welcome. Any comments or suggestions are very welcome.

### CODE REVIEW

**Comment by ackalker** [comment]
Regarding `sound-icons` package: this enhances `speech-dispatcher` by adding 'sound icons', a kind of [earcon](https://en.wikipedia.org/wiki/Earcon)s which are specifically for use with text-to-speech software. `speech-dispatcher` can use these sound icons 'out-of-the-box' as packaged, no change in configuration files necessary.
Regarding `dex`: although it doesn't need compilation as required by `Manual.md`, I think it is useful for Void Linux users who use minimalist window managers or desktop environments, because it allows one to easily run 'autostart' desktop applications from `*.desktop` files in `/etc/xdg/autostart` and `${HOME}/.config/autostart` directories:
```sh
$ dex -a # Run everything autostartable
$ dex -ae i3 # Do not run any DE-specific file unless it specifies `OnlyShowIn=i3`
# dex -ae GNOME # Fake running GNOME desktop
```

**Comment by ebfe** [comment]
Could you rebase this onto master? That should fix the travis builds/results.

**Comment by ackalker** [comment]
@ebfe I've rebased to master and pushed again, thanks.

**Comment by ebfe** [comment]
Installed /usr/bin/dex has a /usr/bin/python2 shebang but package depends on python3? 

Otherwise looks ok, though i'd vote for `s/Bear/bear/`.

**Comment by ackalker** [comment]
Thank you for your comments.
Regarding Bear's Python dependency, the documentation states that Bear supports Python versions >= 2.7, so I made the package depend on `python`, which in Void Linux is now at version 2.7.12_4. Please let me know if I should make the package depend on `python3` instead, in which case I will also change the shebang.
Regarding the package name `Bear`, I was in doubt about whether to capitalize the name as does the documentation does, or to simply name it `bear` after the name of the script. Since the release tarballs don't have the package name, there's no obvious choice. I'll gladly rename the package to `bear` if that is more to your liking.

**Comment by ebfe** [comment]
> Regarding Bear's Python dependency, the documentation states that Bear
> supports Python versions >= 2.7, ...
The python dependency problem is with `dex` not bear. The package
depends on python3 but there is a hook that rewrites unversioned python
shebangs:

```
=> dex-0.7_1: running pre-pkg hook: 03-rewrite-python-shebang ...
   Shebang converted to '#!/usr/bin/python2': /usr/bin/dex
```

> I'll gladly rename the package to `bear` if that is more to your liking.

Your choice really, i just think uppercase in package names are a bit annoying.

**Comment by ackalker** [comment]
Ah, my fault, I'm so sorry. I'll have a look at the `dex` package tomorrow. BTW, my `/usr/bin/dex` has `#!/usr/bin/env python3` as shebang, as it should (it is Python 3 code).

**Comment by ebfe** [comment]
29c37543cf75436b4c602fd1a0f16a11b3c05539 might have changed the default.

File: srcpkgs/dex/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'dex'
+pkgname=dex
+version=0.7
+revision=1
+noarch=yes
+build_style=gnu-makefile
+make_install_args="MANPREFIX=/usr/share/man"
```
Comment (asergi) [inline srcpkgs/dex/template]: `python_version=3`

**Comment by ackalker** [comment]
@ebfe , @asergi Thanks for your tip and review. I've fixed `dex` package with the change.
I've decided to stick with the name `Bear` for that package, it is the name of the project on Github and is used consistently in the documentation, and several other packages in Void also follow this lead. Also, `xbps-query -Rs bear` has no problem finding the package regardless of whether it's name is capitalized.
Thanks for your time, I hope to be able to provide Void Linux with other useful packages in the future.

**Comment by ackalker** [comment]
Thanks to all :smile:

**Pull request merged**